code/modules/client/preference/preferences.dm

```diff
@@ -200,12 +200,20 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	var/list/gear = list()
 	var/gear_tab = "General"
 
+	var/list/custom_names = list("human", "clown", "mime", "ai", "cyborg", "religion", "deity")
```

Comment:
don't forget the human name. These are "backup humans" for like...events and crap.

Reply:
See i forgot to remove that...Do we even need that?

Reply:
we'll need it once lavaland goes in, so might as well put it in now.